packages/@react-spectrum/card/stories/GalleryCardView.stories.tsx

```diff
@@ -78,6 +78,7 @@ let itemsNoThinImages = [
   {width: 1516, height: 1009, src: 'https://i.imgur.com/1nScMIH.jpg', id: 21, title: 'Bob 5'}
 ];
 
+// TODO: accessibility failures regarding article element with role="gridcell", will need to double check when we pick CardView back up
```

Comment:
I let failures like these and the TableView ones (e.g. https://github.com/adobe/react-spectrum/wiki/Known-accessibility-false-positives#tableview) continue to fail so that we don't forget about them. Alternatively, I could mark them as `reviewOnFail: true,` in the config which will no longer mark them as failures, instead moving them to the "incomplete" part of the Accessibility tab:
<img width="338" alt="image" src="https://github.com/adobe/react-spectrum/assets/9661127/6a5119a9-6d93-4fd0-a73e-148541ac517a">
  
This would cut the number of failures down even more, but I'm still worried we'd forget about these errors.